tests/unit/ctest.h

```diff
@@ -79,14 +79,24 @@ struct ctest {
    unsigned int magic;
 };
 
+/*
+ * Message verbosity levels:  Modeled on syslog's severity levels.
+ * See: https://en.wikipedia.org/wiki/Syslog
+ */
+typedef enum msg_level {
```

Comment:
I considered the option of externalizing these by adding them to `include/splinterdb/platform_linux/public_platform.h`, but given that this is still preliminary code, and still cooking ... I figured it's best to scope these levels only to CTest code. For now.

Once we get a better handle on how all this plays out ... we could look into formalizing these levels in an external `.h` file.